cmd/root.go

```diff
@@ -26,7 +26,8 @@ var rootCmd = &cobra.Command{
 			return
 		}
 		if flagVal {
-			fmt.Println("I like these words better!")
+			fmt.Println("I like words and stuff")
+			//https://stackoverflow.com/questions/53879154/println-vs-printf-vs-print-in-go
```

Comment:
Seriously? why is there a stack overflow comment in this PR?? please remove!

Reply:
Obviously, it's for educational purposes.